src/App.tsx

```diff
@@ -14,6 +15,7 @@ function App() {
         <Routes>
           <Route path={routes.PUBLIC.HOME} element={<Home />} />
           <Route path={routes.PUBLIC.FEED} element={<Feed />} />
+          <Route path="/article/:id" element={<FullArticle />} />
```

Comment:
we need to add a route in routes.ts for /article and then paste here like path={`${routes.PUBLIC.ARTICLES}/:id`}